docs/14.0.0-calendar-accessibility-migration.md

```diff
@@ -0,0 +1,162 @@
+  startDateSelectedState: I18n.translate('CALENDAR_RANGE_START_DATE_SELECTED_STATE_LABEL'),
+
+  // CALENDAR_RANGE_END_DATE_SELECTED_STATE_LABEL should be
+  // a string that is read out to screen reader users
+  // when the selected end date has focus.
+  //
+  // English example: "Selected as return date"
+  endDateSelectedState: I18n.translate('CALENDAR_RANGE_END_DATE_SELECTED_STATE_LABEL'),
+
+  // CALENDAR_RANGE_END_AND_START_DATE_SELECTED_STATE_LABEL should be
+  // a string that is read out to screen reader users
+  // when the selected end date is the same as the start date and
+  // this date has focus.
+  //
+  // English example: "Selected as both departure and return date"
+  endAndStartDateSelectedState: I18n.translate('CALENDAR_RANGE_END_AND_START_DATE_SELECTED_STATE_LABEL'),
+
+  // CALENDAR_RANGE_DATE_BETWEEN_START_AND_END_SELECTED_STATE_LABEL
+  // should be a string that is read out to screen reader users
+  // when a date between the start and end date has focus. e.g.
+  // when the start date is 01/03/2021 and the end date is 05/03/2021
+  // and the date with focus is 03/03/2021.
+  //
+  // English example: "Selected between departure and return date"
+  dateBetweenStartAndEndSelectedState: I18n.translate('CALENDAR_RANGE_DATE_BETWEEN_START_AND_END_SELECTED_STATE_LABEL'),
+
+  // CALENDAR_RANGE_NEXT_SELECTION_PROMPT_LABEL should be
+  // a prompt that is read out to screen reader users
+  // when a start date is selected. Should explain to the user
+  // that they should select an end date for the range.
+  //
+  // English example: "Now select a return date (if required)"
+  makeNextSelectionPrompt: I18n.translate('CALENDAR_RANGE_NEXT_SELECTION_PROMPT_LABEL'),
+});
+
+const MyCalendar = (props) => <BpkCalendar selectionType={rangeSelection} {...rest} />;
+
+```
+
+## Multiple Selection
+
+
+### Before
+
+```javascript
+import React, { Component } from 'react';
+import BpkCalendar, { SELECTION_TYPES } from 'backpack-react-native/bpk-component-calendar';
+
+const MyCalendar = (props) => <BpkCalendar selectionType={SELECTION_TYPES.multiple} {...rest} />;
+
+```
+
+### After
+
+```javascript
+import React, { Component } from 'react';
+import BpkCalendar, { makeMultipleSelection } from 'backpack-react-native/bpk-component-calendar';
+
+const multipleSelection = makeMultipleSelection({
+
+  // `CALENDAR_SINGLE_SELECT_HINT_LABEL` should be a string that provides a
+  // hint to the user on how to select a date.
+  //
+  // English example: "Double tap to select date"
+  selectHint: I18n.translate('CALENDAR_SINGLE_SELECT_HINT_LABEL'),
+
+  // `CALENDAR_SINGLE_DESELECT_HINT_LABEL` should be a string that provides a
+  // hint to the user on how to deselect a selected date.
+  //
+  // English example: "Double tap to deselect date"
+  selectHint: I18n.translate('CALENDAR_SINGLE_DESELECT_HINT_LABEL'),
+});
+
+const MyCalendar = (props) => <BpkCalendar selectionType={multipleSelection} {...rest} />;
+
+```
+
```

Comment:
No comment other than to say this migration guide is exceptionally well written! Nice work 🎉